.travis.yml

```diff
@@ -32,4 +32,4 @@ script:
     - cargo build # Linter is also executed here
     - RUST_BACKTRACE=1 cargo test
     - jshint static/js/*.js test/selenium/*.js
-    - (cargo run -- -n $BOX_HOST_NAME -p $BOX_PORT &> /dev/null &) ; npm test
+    - (cargo run -- -n $BOX_HOST_NAME -p $BOX_PORT
```

Comment:
My apologies, the line should be: `(cargo run -- -n $BOX_HOST_NAME -p $BOX_PORT &) ; npm test`.
* `&> /dev/null` means redirect the standard and the error output to nothing.
* `&` before `)` means we start a new process
* `;` means we continue to execute the next command, whatever the result of the first command was
* `npm test` runs the Selenium tests